studio/src/pages/RequestorPage/AddRouteToCollection/AddToRouteForm.tsx

```diff
@@ -0,0 +1,49 @@
+import { useCollections } from "@/queries";
+import { useStudioStore } from "../store";
+import { AddToRouteFormItem } from "./AddToRouteFormItem";
+
+type Props = {
+  onSuccess: () => void;
+};
+
+export function AddToRouteForm(props: Props) {
+  const { data: collections, isLoading } = useCollections();
+  const { activeRoute } = useStudioStore("activeRoute");
+
+  // No active route? Then there's nothing to do here
```

Comment:
could be good to at least put a message in the UI here — otherwise if the user has typed something that doesn't match the routes, then all they see is an empty popover 

<img width="742" alt="image" src="https://github.com/user-attachments/assets/3dfddeb2-4729-4a25-a685-ea5ad59cbe0b" />